addons/spark/0.x/spark-1.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: spark
+  labels:
+    kubeaddons.mesosphere.io/name: spark
+    kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
+  annotations:
+    stage.kubeaddons.mesosphere.io/spark: Beta
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.2.0-1"
+    appversion.kubeaddons.mesosphere.io/spark: "0.2.0"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.15.0
+  cloudProvider:
+    - name: aws
+      enabled: true
+    - name: azure
+      enabled: true
+    - name: gcp
+      enabled: true
+    - name: docker
+      enabled: true
+    - name: none
+      enabled: true
+  requires:
```

Comment:
out of curiosity, shouldn't KUDO be listed as a requirement too?

Reply:
According to this PR https://github.com/mesosphere/kubeaddons/pull/698, KUDO became a part of the controller setup, so it will be present by default

Reply:
:point_up: